pkg/kv/kvtest/store.go

```diff
@@ -369,3 +374,327 @@ func testScanPrefix(t *testing.T, ms MakeStore) {
 		t.Fatal("ScanPrefix entries didn't match:", diff)
 	}
 }
+
+func testDeleteWhileIterPrefix(t *testing.T, ms MakeStore) {
+	// iteration
```

Comment:
Strange comments and formatting ??

Reply:
That's just the way the different `iteration` vs `deletion` orders are depicted in the issue

Reply:
Got it now 🤟